### ISSUE TITLE
Rename default "master" branch to "main"

### ISSUE THREAD

**Issue by fnogatz** [impress/impress.js#842]
I would like to suggest the renaming of the project's default branch `master` to `main`, following Git's defaults. A brief summary of the recent developments regarding the shift towards `main` instead of `master` is given in these articles of [GitLab](https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/) and [GitHub](https://github.com/github/renaming). I think only @henrikingo currently has the permissions to do so. Any chances we can change this?

**Comment by henrikingo** [comment]
Yes, I agree now is a good time. I would propose if you finish some of your other work first, I will add you to the admin group, so that you can do this task yourself. I think mainly you should then double check that the CI still works correctly.
